test/jest/domain.test.js

```diff
@@ -0,0 +1,128 @@
+/* @flow */
+import {
+    getPayPalDomain,
+    getPayPalAPIDomain,
+    insertMockSDKScript
+} from '../../src';
+
+
+beforeEach(() => {
+    __ENV__ = 'test';
+});
+
+describe(`getPayPalDomain function test cases`, () => {
+    it('should successfully get the production paypal domain', () => {
+        __ENV__ = 'production';
+        const expectedPayPalDomain = 'https://www.paypal.com';
+
+        if (getPayPalDomain() !== expectedPayPalDomain) {
+            throw new Error(`Expected paypal domain to be ${ expectedPayPalDomain }, got ${ getPayPalDomain() }`);
+        }
+    });
+
+    it('should successfully get the stage paypal domain', () => {
```

Comment:
This is minor but what do you think about this grouping for the tests? It consolidates the tests more and uses `test` instead of `it` to stay consistent with the server-side jest tests.

```js
describe('getPayPalDomain()', () => {
    test('should get the expected paypal domain for each environment', () => {
        __ENV__ = 'production';
        expect(getPayPalDomain()).toBe('https://www.paypal.com');

        __ENV__ = 'sandbox';
        expect(getPayPalDomain()).toBe('https://www.sandbox.paypal.com');

        __ENV__ = 'stage';
        expect(getPayPalDomain()).toBe('https://msmaster.qa.paypal.com');
    });

    test('should get the expected paypal domain when using query param env=sandbox_v2', () => {
        insertMockSDKScript({
            query: {
                'client-id': 'sb',
                'env':       'sandbox_v2'
            }
        });

        __ENV__ = 'sandbox';
        expect(getPayPalDomain()).toBe('https://www-v2.sandbox.paypal.com');
    });

    test('should throw an error when the environment is not found', () => {
        __ENV__ = 'invalid_environment';
        expect(getPayPalDomain()).toThrow(`Can not get paypal domain for env: ${ __ENV__ }`);
    });

})

```